src/components/Map/Markers/PoiMarkers/index.tsx

```diff
@@ -28,6 +28,7 @@ const PoiMarkers: React.FC = () => {
     if (pois && Object.keys(pois).length !== 0) {
       markers.poi.setPois(pois);
       setOnPoiMarkerClick(handleClick);
```

Comment:
Please achieve this feature by editing `src/components/Map/Markers/ClusterMarkers/index.tsx` instead.
You may want to add `isCurrentSearchParamsCluster` which is similar to `isCurrentSearchParamsPoi`.

```suggestion
```

Reply:
done, clear cluster at `src/components/Map/Markers/ClusterMarkers/index.tsx`